packaging/common/rabbitmq-server-ha.ocf

```diff
@@ -0,0 +1,1585 @@
+#!/bin/bash
```

Comment:
We have much bashisms in this script, I suggest to address refactoring to the "sh" by separate changes. Hopefully once we would have tests and maybe CI for this script as well